internal/directory/provider.go

```diff
@@ -73,6 +75,7 @@ func GetProvider(options Options) (provider Provider) {
 				auth0.WithDomain(options.ProviderURL),
 				auth0.WithServiceAccount(serviceAccount))
 		}
+		errSyncDisabled = fmt.Errorf("invalid auth0 service account: %w", err)
```

Comment:
We could use `multierror` package here?

Reply:
there aren't multiple errors here. it's still one error.